client/src/pages/SignUp/SignUp.tsx

```diff
@@ -28,6 +29,12 @@ export default function Register(): JSX.Element {
         updateSnackBarMessage(data.error.message);
       } else if (data.success) {
         updateLoginContext(data.success);
+        const shoppingUrl = 'https://team-granny-smith-s3.s3.ca-central-1.amazonaws.com/default-images/Shopping.jpg';
```

Comment:
Would it make more sense for this logic to be performed entirely on the backend when you hit the signup route?

Reply:
Sure, I'll get this changed